ovmf/memory.go

```diff
@@ -77,4 +77,6 @@ type GuestPhysicalRegion struct {
 type MaterialGuestPhysicalRegion struct {
 	GPR        GuestPhysicalRegion
 	HostBuffer []byte
+	// Attributes is a TDVF-only field to track whether the buffer is measured.
```

Comment:
This comment suggests that it's a boolean that should be named IsBufferMeasured. Can you clarify how it tracks whether the buffer is measured?

Reply:
Updated.